docs/en/stage1.md

```diff
@@ -0,0 +1,99 @@
+
+## Schedule and assignment list
+https://docs.google.com/spreadsheets/d/1c1mN27MqQz7T8DcD_tMwkOSbizu3VFHXGf0eXwW7WGw/edit#gid=0
+
+## Lecture videos
+   - Records of webinars and offline lectures can be found on our channel - https://www.youtube.com/user/therollingscopes
+   - Most lectures are grouped by playlists - https://www.youtube.com/user/therollingscopes/playlists
+   - Records of an introductory session of past sets:
+       - 2018Q3 https://youtu.be/Gp0P4LP9QOg
+       - 2018Q1 https://youtu.be/fqZVq-I2Zt4
+       - 2017Q3 https://youtu.be/ch5vXi45ttg
+   - On the Imaguru channel
+   https://www.youtube.com/channel/UCiJcyyoWidR2gYmwSy0yyPQ
+  
+  The video is better to watch at speeds of 1.25 or 1.5
+
+## Types of classes
+### Additional offline lectures
+Offline Stream is a set of face-to-face lectures. For which we usually register up to 70 people. The quantity and material of the lecture depends on the teacher, but correlates with the material of webinars and lectures offered for self-study.
+Information about additional offline classes in your city we will post in the channel #announcements
+as well as at https://community-z.com/communities/the-rolling-scopes/events
+Participation is welcome but not required.
+
+### Self-study
+The schedule has lectures marked as "Self-study." It is expected that you have looked/look at the recordings of these lectures. It’s more convenient to watch at a speed of 1.25+
+
+### HTML/CSS Essentials Test
+- Test format - Google form, in which you will need to enter/select the correct answers. A link to the Google form will be in the announcements chat.
+- The test consists of 45 questions per 100 points.
+- The test is carried out 2 times. The maximum score of two tests is taken into account.
+- The test takes 24 hours.
+- It is expected that before the test you will be able to watch lectures on HTML/CSS (links are in the schedule), as well as go through tasks on codeacademy
+
+### CodeJam
+This is a short task whose description is not known in advance, and a limited time is allocated for execution (from 60 minutes to 48 hours).
+For example, on Monday at 21:00 everyone gets a link with a task, which takes 180 minutes to complete.
+
+### Algorithmic tasks
+- Sample assignment - https://github.com/SIARHEI-SMANTSAR/love-triangle
+- Estimates for these tasks are summed up in the overall score with a coefficient from 0.1 to 0.4 (maximum you can get 40 points for solving one problem)
+- We do not require solving all problems. In the last set, people who solved only 2-3 problems passed to the second stage of training.
+- Schedule tasks ⇒ deductions. Think carefully before submitting someone else’s code to get 10 points.
+- If during the interview you do not know how you solved the assignment you passed ⇒ there was a write-off ⇒ expulsion.
+- If during the interview you know how to solve the task, but you cannot solve the obviously simpler task ⇒ there was a cancellation ⇒ expulsion.
```

Comment:
- If during the interview you know how to solve a task that you submitted, but you cannot solve an easier task ⇒ that means you plagiarized ⇒ and that means you will be expelled.